custom_components/tuya_local/devices/ips_pro_heatpump.yaml

```diff
@@ -35,34 +42,9 @@ primary_entity:
           value: F
         - dps_val: true
           value: C
-    - id: 105
-      name: hvac_action
-      type: string
-      mapping:
-        - dps_val: warm
-          constraint: hvac_mode
-          conditions:
-            - dps_val: false
-              value: "off"
-            - dps_val: true
-              value: idle
     - id: 106
       name: temperature
       type: integer
```

Comment:
temperature requires a range.  I'm not sure why you removed this.  If it is because the temperature remains in C even when the setting is for F, then maybe it needs to be solved by a redirect to another dp (temp_set_f), or by hardcoding the unit to C and ignoring temperature_unit.

Reply:
I have removed it because the range is handled by min and max temperature. The values there change dependent on the mode (cool/auto/heat)

Let me know if you don't want me to contribute. I can use my fork as well ;)

Reply:
Yes, you're right. I actually  noticed that the first time I reviewed, but forgot in the interim